patch-package
--- a/node_modules/@huggingface/inference/src/utils/getEndpoint.ts
+++ b/node_modules/@huggingface/inference/src/utils/getEndpoint.ts
@@ -23,7 +23,7 @@
 		? request.model
 		: (request as any).repo_id;
 
-	const endpoint = `https://api-inference.huggingface.co/${type}/${model}`;
+	const endpoint = `https://router.huggingface.co/hf-inference/${type}/${model}`;
 
 	if (type === "models" && !endpoint.startsWith("https://api-inference.huggingface.co/models/")) {
 		// prevent security issue
